src/Chromosome.jl

```diff
@@ -59,10 +62,8 @@ function print_chromosome(c::Chromosome, active_only::Bool=false)
     end
 
     for i = 1:length(c.outputs)
-        active = c.outputs[i].active ? "+" : "*"
-        if c.outputs[i].active || !active_only
-            print("[",c.outputs[i].input,"out",i,active,"] ")
-        end
+        active = "+"
```

Comment:
We know that all output nodes are active.  However, now I don't think that "print_chromosome" should be changing the active status of nodes.  So I don't think this change should be done.